doc/download.rst

```diff
@@ -40,15 +40,13 @@ Live media is available in the OVA format, and is suitable, for example, for imp
 If you run **Windows**, this is the option for you OR if you just want to run NEST without installing it on your computer.
 After downloading the virtual machine, check out the :doc:`install instructions for Live Media <installation/livemedia>`.
 
-* `NEST Live Media 2.18.0 <https://nest-simulator.org/downloads/gplreleases/lubuntu-18.04_nest-2.18.0.ova>`_
-
-  `Checksum 2.18.0 <https://nest-simulator.org/downloads/gplreleases/lubuntu-18.04_nest-2.18.0.ova.sha512sum>`_
+* `NEST Live Media 2.20.1 <https://nest-simulator.org/downloads/gplreleases/lubuntu-18.04_nest-2.20.1.ova>`_
```

Comment:
Maybe you can discuss the term "Live Media" in the docteam meeting? I really hate it and I think that since CDs are not a thing anymore, we are solely referring to "virtual machine images" if we use that term and should rather just use that term instead. From my perspective, this could, however, also be handled in another PR. Just sayin'.

Reply:
> Maybe you can discuss the term "Live Media" in the docteam meeting? I really hate it and I think that since CDs are not a thing anymore, we are solely referring to "virtual machine images" if we use that term and should rather just use that term instead. From my perspective, this could, however, also be handled in another PR. Just sayin'.

@sarakonradi What do you think? I don't really know where the term "live media" comes from. It has been around for quite some time.

Reply:
The old term was introduced by me somewhere around 2004, when the new big thing were _Live CDs_ or _bootable USB drives_. The unifying term for the ISO images usable on both types of media was officially _Live Meda_. Being young and naïve, I thoughtlessly took that over, but think it is about time to drop it.